README.md

```diff
@@ -111,6 +111,32 @@ make build-cli # build cli tool
 --checkpoint-storage-location-override <hplStorageFileOverride> \
 ```
 
+balances: show current on-chain balances for USDC, gas token, and any custom assets requested
+
+```shell
+./build/solvercli balances \
```

Comment:
now that the cli is getting a lot more functionality, i think we should provide instructions for users to build the cli and then add it to their path, then they can just run `solver balances ...` instead of `./build/solvercli balances ...` (and maybe rename from solvercli to just solver) (this could also be in another pr, but imo it will feel more like a polished cli then)

Reply:
good shout, done